packages/components/src/TemporaryAddressChange/Component.purs

```diff
@@ -45,6 +46,7 @@ type Self = React.Self Props State
 
 type Props =
   { subsno    :: Int
+  , cusno     :: String
```

Comment:
Wait, are we really using `Strings` and `Ints` as cusno in our code? 😅

Reply:
https://github.com/KSF-Media/affresco/blob/master/packages/components/src/Api/Subscription.purs#L32
https://github.com/KSF-Media/affresco/blob/master/packages/components/src/Persona.purs#L346

🤷‍♀️🤷‍♀️🤷‍♀️🤷‍♀️🤷‍♀️ nice